### PR TITLE
Updated README and fixed broken links

### PR DESCRIPTION
**Pull request by Chris-Schnaufer**
## Description

The README needed updating due to files moving, additional documentation, and clarifications

## Types of changes

Fixed links, changes "repo" to "repository", added some links, and reworked some sentences

## Checklist:

- [x] I have read the documentation on [CONTRIBUTING](https://github.com/AgPipeline/Organization-info) 
- [x] My updates require a change to the documentation 
- [x] I have updated the documentation accordingly
- [ ] I have added tests to cover my changes
- [ ] All new and existing tests passed

## Time frame for completing Pull Request review (from creation date):
- [ ] As soon as possible
- [x] In the next two weeks
- [ ] In the next month

## Focus of Pull Request:
All comments are welcome regardless of which items are checked off.
- [ ] All of the below (no need to check off the other boxes)
- [ ] There are no errors or inconsistencies
- [x] Everything appears clear and easy to understand
- [ ] Suggest improvements
- [ ] Identify areas where functionality and/or documentation is missing

## Hints to reviewers:

N/A

### CODE REVIEW

**Review by kyclark — APPROVED** [review]
Seems fine to me.

File: README.md [hunk 1]
```diff
@@ -40,28 +42,28 @@ Another aspect is to keep a repository's meaning clear; for example. don't combi
...
-Keeping repos as single purpose as possible is also encouraged.
-This does not prevent major complexity within a repo.
+Keeping repositories as single purpose as possible is also encouraged.
+This does not prevent major complexity within a repository.
 
 In the end, use your best judgement, don't be afraid to reorganize, and ask if you have any questions!
 
 # Issues
 
 We are run by the issues that are entered into our system.
```
Comment (MagicMilly) [inline README.md]: I know this isn't included in the PR, but this sentence is unclear to me. If I'm understanding correctly, would something like 
```suggestion
Our work is directed by the issues submitted.
```
make more sense?

Comment (Chris-Schnaufer) [reply README.md]: Definitely more sense has been made! Thanks!

**Review by MagicMilly — APPROVED** [review]
Looks good 👍

**Pull request merged**